### PR TITLE
Make browser launch platform-independent

### LINKED ISSUE #1: First-time usage, CLI hanging during authentication (Linux)
👋 Hi!

I'm excited to give this a shot, but I'm having trouble getting authentication to work.

I've created an API id and secret in Strava; installed Ruby 2.5.5 and the CLI through rubygems, but any attempt at authentication hangs after entering my credentials:

Here's where it hangs:

```
matt@penguin:~$ strava run athlete
[2020-05-31 13:00:42] INFO  WEBrick 1.4.2
[2020-05-31 13:00:42] INFO  ruby 2.5.5 (2019-03-15) [aarch64-linux-gnu]
Enter Strava client ID: *****
Enter Strava client secret: ****************************************
[2020-05-31 13:00:52] INFO  WEBrick::HTTPServer#start: pid=12884 port=4242
```

Hitting ctrl+c results in an `^Cerror: Authorization Error` message.

I've tried this on a few Linux workstations (WSL2 Linux container running Debian 10, a "native" Ubuntu 20.04, and a Chromebook crostini container on Debian 10); all with the same result.

Have you seen this error before, maybe I'm missing something in the setup process?

### PR DESCRIPTION
**Pull request by mattdsteele**
Fixes #1

I was able to build and install it locally and it appears to work on both vanilla Linux and WSL, but this probably needs a second set of eyes. I'm not very familiar with Gem packaging, so let me know if anything needs changed.  I couldn't get specs to run; it timed out for me for some reason.

As info, there's still a workaround required for WSL: https://github.com/copiousfreetime/launchy/issues/117#issuecomment-363277558

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,6 +2,8 @@ source 'http://rubygems.org'
 
 gemspec
 
+gem 'launchy'
```
Comment (dblock) [inline Gemfile]: This should only go into `.gemspec`, so that it gets installed with `bundle install`.

File: lib/cli/auth.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
+require 'launchy'
```
Comment (dblock) [inline lib/cli/auth.rb]: Shouldn't be needed and still work because of .gemspec.

Comment (dblock) [inline lib/cli/auth.rb]: Shouldn't be needed. Otherwise add to  https://github.com/dblock/strava-ruby-cli/blob/master/lib/strava-ruby-cli.rb

**Review by dblock — CHANGES_REQUESTED** [review]
Thanks! Almost there. Run `rubocop -a ; rubocop --auto-gen-config` to auto-fix some ruby, and double check that `rake` succeeds. Add a line to CHANGELOG.

File: lib/cli/auth.rb (hunk 2 above)
Comment (mattdsteele) [reply lib/cli/auth.rb]: I got errors without the require, so I moved it to `lib/strava-ruby-cli.rb`

**Comment by dblock** [comment]
Thanks, merged.

Is there an error when the browser fails to launch (e.g. without $BROWSER), or do we get a hang again? Want to check out https://github.com/dblock/strava-ruby-cli/issues/3?

**Pull request merged**